Polyhedron/demo/Polyhedron/Scene_surface_mesh_item.cpp

```diff
@@ -815,7 +815,9 @@ Scene_surface_mesh_item_priv::triangulate_facet(face_descriptor fd,
     diagonal = item->diagonalBbox();
   else
     diagonal = 0.0;
-  FT triangulation(fd,normal,smesh_,diagonal);
+  const qglviewer::Vec off = static_cast<CGAL::Three::Viewer_interface*>(QGLViewer::QGLViewerPool().first())->offset();
```

Comment:
@lrineau This does not play right with #2967 in integration. I think the two PRs shouldn't be tested together.